kotlin/internal/jvm/impl.bzl

```diff
@@ -419,7 +419,7 @@ def kt_compiler_plugin_impl(ctx):
 
     return [
         DefaultInfo(files = classpath),
-        info,
+        java_common.merge([]),
```

Comment:
What consumes this JavaInfo?

Reply:
I couldn't find anything

Reply:
Let's remove it, then.

Reply:
sorry, updated

Reply:
Looking closer into it, I see it's referenced in
https://sourcegraph.com/github.com/bazelbuild/rules_kotlin/-/blob/kotlin/internal/jvm/compile.bzl?L707#tab=history
https://sourcegraph.com/github.com/bazelbuild/rules_kotlin/-/blob/kotlin/internal/jvm/plugins.bzl?L51

Do we still want to remove it?

Reply:
Yes -- providing an empty JavaInfo doesn't help, and increases the chances of something stupid happening.

Speaking of stupid... why does the target have a `KtJvmPluginInfo` and no `JavaInfo`? It isn't due to `KtCompilerPluginInfo`.

Maybe it's an export issue?